locales/pt_BR/LC_MESSAGES/duckduckgo.po

```diff
@@ -2611,7 +2550,7 @@ msgid "No %s found for %s%s%s."
 msgstr "Nenhum %s encontrado para %s%s%s."
 
 msgid "No More Photos"
-msgstr "Sem Mais Fotos"
+msgstr "Remover Fotos"
```

Comment:
The previous translation seems more accurate. This should be shown when there are no more photos to show, it's not supposed to be a verb.

```suggestion
msgstr "Sem Mais Fotos"
```